### DISCUSSION TITLE
The color of the population simulation in population compasion

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1510] [Questions & Problems]
Hello, I want to use PK-sim to compare the established groups, I found that I chose the color, but the final color is not the one I want, and there are only two options to set the color, when I have four groups, can I only use the default color generated by the software? Can I make a change?
![2](https://user-images.githubusercontent.com/95362728/233922985-83ab321e-425a-4f96-bbfa-3a54f89aac94.jpg)
<img width="338" alt="3" src="https://user-images.githubusercontent.com/95362728/233922992-e80b3f49-408a-4abd-9d95-07632424c31a.png">
<img width="415" alt="4" src="https://user-images.githubusercontent.com/95362728/233922997-2f5d7632-d222-4d76-8906-40ef93d9597d.png">
![微信截图_20230320093829](https://user-images.githubusercontent.com/95362728/233923000-62357dca-677b-4ff1-b1b7-5b5aa4e27ebf.jpg)

**Comment by Yuri05** [comment]
In the comparison, you could create **grouping by the simulation name** and assign colour to every simulation
(compare the attached example [POP_Comparison.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/11313862/POP_Comparison.zip))

![grafik](https://user-images.githubusercontent.com/25061876/234091429-90e644ca-ce1c-40dc-b79c-629ba5632277.png)

Then in the time profile settings view, **remove the simulation name first** and add you group:

![grafik](https://user-images.githubusercontent.com/25061876/234091977-d4dda3e2-2625-408c-bddd-1b3f46798dc5.png)

Then you will get the colours you have defined in the grouping

![grafik](https://user-images.githubusercontent.com/25061876/234092143-0fb4bfed-afeb-4515-9dc8-fe18ac044dbb.png)

**Reply by WTY-tt** [reply]
Ok, thank you for your guidance, I know what to do, thank you very much!
